src/wasm.h

```diff
@@ -1332,9 +1332,26 @@ class ArrayLen : public SpecificExpression<Expression::ArrayLenId> {
   void finalize() { WASM_UNREACHABLE("TODO (gc): array.len"); }
 };
 
+// A named element whose name can end up in the name section.
+// Explicit names are ones that we read from the input file and
+// will be written the name section in the output file.
+// Implicit names are names that binaryen generated for internal
+// use only and will not be written the name section.
+struct Namable {
```

Comment:
`Namable` => `Nameable`

Also, do we plan to use this for anything that's not `Importable`? If so, it might make sense to just make this interface part of `Importable`.

Reply:
Good point..  went I wrote this I had assumable there would be other nameable thing but I guess not yet.

We probably want to rename `Importable` to something like `External` (i.e. element that can be imported and/or exports)  to match the spec language and that used in the wasm C API, but I'll leave it for now.